monitoring/mock_uss/scdsc/database.py

```diff
@@ -10,7 +10,7 @@
 class FlightRecord(ImplicitDict):
     """Representation of a flight in a USS"""
     op_intent_injection: scd_injection_api.OperationalIntentTestInjection
-    flight_authorisation: scd_injection_api.FlightAuthorisationData
+    flight_authorisation: Optional[scd_injection_api.FlightAuthorisationData]
```

Comment:
nit: I'd make this non-optional in accordance with [this comment](https://github.com/interuss/dss/pull/710#discussion_r806115352)